hooks/terraform_validate.sh

```diff
@@ -113,7 +113,7 @@ function per_dir_hook_unique_part {
   # First try `terraform validate` with the hope that all deps are
   # pre-installed. That is needed for cases when `.terraform/modules`
   # or `.terraform/providers` missed AND that is expected.
-  terraform validate "${args[@]}" 2>&1 && {
+  terraform validate "${args[@]}" > /dev/null 2>&1 && {
```

Comment:
Redirecting both stderr and stdout can be accomplished via single operator.
```suggestion
  terraform validate "${args[@]}" &> /dev/null && {
```
What I'm concerned with is that with this change we hide tf validate output in case when it fails, which doesn't seem to be what's expected.
So probably we need to assign output to a var and echo it when tf validate fails? Sort of this:
```suggestion
  local __validate_output
  __validate_output=$(terraform validate "${args[@]}" 2>&1) && {
    exit_code=$?
    return $exit_code
  } || {
    common::colorify "yellow" "$__validate_output"
  }
```
Apologies if I'm missing context and this is expected to discard tf validate failure message.

Reply:
Check https://github.com/antonbabenko/pre-commit-terraform/issues/557#issuecomment-1755269452

Reply:
That check is used only for successful validation. 

And there ~zero logs for that case